.travis.yml

```diff
@@ -93,6 +95,20 @@ matrix:
         # Report success since we have overridden default behaviour
         - bash -c "$STATUS" success "Local $NAME testing has passed"
 
+    - env:
+        - TEST="astyle"
+      install:
+        - sudo apt-get install astyle
+      script:
+        - export ARTISTIC_STYLE_OPTIONS=".astylerc" && $ find -regex '.*\.\(h\|c\|hpp\|cpp\)' -type f ! -path "./cmsis/*" ! -path "./features/mbedtls/*"  ! -path "./features/FEATURE_LWIP/lwip/*"  ! -path "./rtos/TARGET_CORTEX/rtx5/RTX/*"  ! -path "./BUILD/*"  ! -path "./tools/*" | xargs -n 100 -I {} bash -c "astyle --options=.astylerc \"{}\"" > astyle.out
```

Comment:
@geky what magic can we use to exclude some directories ? I defined only some generic ones, but lot of targets contain their own drivers.

I tried to use --exclude, but always end up with error unmatched pattern from astyle 😢 

Shall we define exclude paths in a file, or provide .astyleignore file in those folders?